relion/viewers/viewer_ctfrefine.py

```diff
@@ -170,23 +175,47 @@ def _displayDefocus(self, e=None):
 
         self.show()
 
+    def _showImages(self, *imgs, **kwargs):
+        title = kwargs.get('title', '')
+        if self.useMatplotlib:
+            return self._showImagesMatplotlib(title, *imgs)
+        else:
+            return [DataView(img) for img in imgs]
+
+    def _showImagesMatplotlib(self, title, *imgs):
+        import mrcfile
```

Comment:
for this we'll need to include it into pip requirements?

Reply:
I wasn't sure if we were already using mrcfile somewhere...we could also read the image with ImageHandler class (xmipp under the hood) and use the getData() that will give us the numpy array...probably the best solution right now....anyway this import should not stay there.

Reply:
it's still there.. :)

Reply:
Yes, I will change this, please keep checking and I will fix this before the merge.

Reply:
Done now with ImageHandler